src/graphics/C4Shader.h

```diff
@@ -210,6 +210,10 @@ class C4ShaderCall
 		if (pShader->HaveUniform(iUniform))
 			glUniform1i(pShader->GetUniform(iUniform), iX);
 	}
+	void SetUniform2i(int iUniform, int iX, int iY) const {
+		if (pShader->HaveUniform(iUniform))
+			glUniform2i(pShader->GetUniform(iUniform), iX, iY);
+	}
```

Comment:
The function is not used currently, but the same holds true for SetUniform1iv and others. I guess it's okay to leave it there right now 🤞